proto/s2s.go

```diff
@@ -20,6 +20,13 @@ type Job struct {
 	Data      map[string]interface{} `json:"data"`      // job-specific data during Job.Run
 	Retry     uint                   `json:"retry"`     // retry N times if first run fails
 	RetryWait uint                   `json:"retryWait"` // wait time (milliseconds) between retries
+	Sequence  *Sequence              `json:"sequence"`
+}
+
+type Sequence struct {
```

Comment:
`// Sequence represents ...`

Code docs more lax deeper in the code we go, but here at the very surface we need the best possible docs we can muster.

Reply:
Added a comment. TBH I was just writing code and forgot to come back and update comments.